ClimsoftVer4/ClimsoftVer4/formMetadata.vb

```diff
@@ -1836,6 +1835,53 @@ Err:
         txtClosingDate.Text = ClosingDate.Text
     End Sub
 
+    Private Function DMSToDD(Direction As Char, Deg As String, Min As String, Sec As String) As String
+        ' Convert value in Degrees, Minutes and Seconds (DMS) to Decimal Degrees (DD)
+        ' Direction must be N, S, E or W
+        If IsNumeric(Deg) And IsNumeric(Min) And IsNumeric(Sec) And Not Direction = vbNullChar Then
```

Comment:
In addition to your checks that the Deg, Min and Sec values are numeric, I've added a check that the Direction is not empty (Null)

Reply:
@isedwards - Very good